.changeset/curly-snails-rush.md

```diff
@@ -0,0 +1,9 @@
+---
+'@web/test-runner-puppeteer': minor
+'@web/test-runner-chrome': minor
+'@web/dev-server-hmr': minor
+'@web/browser-logs': minor
+'@web/dev-server': minor
```

Comment:
I think you need to bump only

```
'@web/test-runner-puppeteer': minor
'@web/test-runner-chrome': minor
```

?

other packages don't have changes in the API, you only updated the devDependencies

Reply:
Update the changeset :smile: